yang/vyatta-protocols-frr-zebra-v1.yang

```diff
@@ -0,0 +1,227 @@
+module vyatta-protocols-frr-zebra-v1 {
```

Comment:
Generally the names used in the YANG should not be tied to the implementation. Call this something other than zebra, maybe something like link-params.